pom.xml

```diff
@@ -559,8 +559,8 @@
                 <configuration>
                     <skipCheckoutIfExists>false</skipCheckoutIfExists>
                     <connectionUrl>scm:git:https://github.com/pagopa/pagopa-ecommerce-commons.git</connectionUrl>
-                    <scmVersionType>tag</scmVersionType>
-                    <scmVersion>${pagopa-ecommerce-commons.version}</scmVersion>
+                    <scmVersionType>branch</scmVersionType>
```

Comment:
Bump to next ecommerce-common version

Reply:
Solved by https://github.com/pagopa/pagopa-ecommerce-transactions-service/pull/558/commits/be7f777206043f365ad0f02b16691aa6662e39a1